### ISSUE TITLE
rsign conversion error

### ISSUE THREAD

**Issue by syrusakbary** [wasmerio/wasmer-js#30]
The shell can't run `rsign` because some issue when transforming:

```bash
$ rsign
[INFO] Doing Transformations for "rsign"
wapm shell: parse error (CompileError: WebAssembly.compile(): section was shorter than expected size (648008 bytes expected, 647826 decoded) @+650103)
```

**Comment by torch2424** [comment]
Also, clang and viu are not working. May be the same issue 😄

**Comment by torch2424** [comment]
This was fixed by #97 . Clang still isn't working, but decided to do that later
